src/md/release.md

```diff
@@ -156,6 +156,14 @@ svn import svn-dist https://dist.apache.org/repos/dist/dev/apex/${RNAME}-RC1 -m
 
 ## Build and Deploy Documentation
 
+### Javadoc
```

Comment:
@vrozov Buildbot needs a branch name parameter at various places; it may work wit a tag but I've never tried it.
@tweise Since there is some ongoing discussion, I'll wait a few more hours and merge if nothing further emerges